spec/features/address_bar_spec.rb

```diff
@@ -10,6 +10,9 @@ def query_string
   context 'when loading collection details page from search page' do
     before :all do
       visit '/search'
+      if page.has_link?('closeInitialTourModal')
```

Comment:
Thank you for this one - I have seen a slowdown and did not know has_link was a trap.  I'll figure out a different method and update these tests.